src/main/resources/reference.conf

```diff
@@ -0,0 +1,8 @@
+latis {
+  about {
+    // The default values for the server's /about endpoint
+    id = "LASP"
+    title = "LASP HAPI Server"
+    contact = "your@email.here"
```

Comment:
We should decide on these default values together.

Reply:
- [ ] ...and decide if we want to put any (commented out?) values in `application.conf` to help users with the syntax. I guess either way we'll need to update the `Configuration` section of the README before we merge this.

Reply:
After thinking about it more, maybe we just don't have defaults because there aren't really reasonable defaults.

For now, that would mean that if they aren't set every request to the `/about` endpoint would result in a 500. I'd prefer that the service just fail to start, but that's probably too much refactoring for now and could be handled in another ticket.

We should at least have an [environment variable override](https://github.com/lightbend/config#optional-system-or-env-variable-overrides), because that's probably how we'll set these in production.

Reply:
Okay, default values have been removed. And I confirmed that we get a 500 if we hit that endpoint without setting a required param.

I added environment variable overrides to `application.conf` too. The naming convention they seemed to lay out in their documentation was `FORCED_<PARAM_NAME>` so I stuck with that. E.g. the override for `id` is `FORCED_ID`.

Note that I didn't add the overrides to `reference.conf` because they don't actually override anything when you put them there. And these will only override other values specified in `application.conf` if those values are written before these overrides (when lightbend sees multiple values for a param in a file it keeps the last one to be declared).

Reply:
I was saying "overrides" because that's what the section with the best example of the syntax was called, but I guess that's misleading because it's only a specific application of this syntax. It's really just looking for an environment variable because it can't resolve the substitution in the configuration, and using `?` allows it to fail for us to deal with later. We don't actually have anything for it to override because the environment variable itself is the default.

Reply:
Okay, I gotcha. Makes sense. And my note about its overriding behavior in `application.conf` is moot anyway because users probably won't be touching that file; our README instructs them to create their own `latis.conf` files rather than edit `application.conf`.